compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -207,11 +210,13 @@ object Completion {
      * considered.
      */
     def addMemberCompletions(qual: Tree)(implicit ctx: Context): Unit = {
-      addAccessibleMembers(qual.tpe)
-      if (!mode.is(Mode.Import)) {
-        // Implicit conversions do not kick in when importing
-        implicitConversionTargets(qual)(ctx.fresh.setExploreTyperState())
-          .foreach(addAccessibleMembers)
+      if (!qual.tpe.widenDealias.isBottomType) {
+        addAccessibleMembers(qual.tpe)
+        if (!mode.is(Mode.Import) && !qual.tpe.isRef(defn.NullClass)) {
+          // Implicit conversions do not kick in when importing
```

Comment:
Ah, this fix makes more sense! 

I wonder why *this* part doesn't work correctly, but looks okay... I'd probably add
```
// and for `NullClass` they produce unapplicable completions (for unclear reasons).
```